locales/sv-se.json

```diff
@@ -0,0 +1,198 @@
+{
+  "button.label.clickMe": "",
+  "button.label.language": "",
+  "button.toastMessage": "",
+  "clicked.times-0": "",
+  "clicked.times-1": "",
+  "clicked.times-2": "",
```

Comment:
 😜

Reply:
Sorry, I didn't understand you.

I copied those from `en.json`, and they did not have a translation there. Should they have a translation?